src/sqlfluff/rules/L036.py

```diff
@@ -82,12 +90,21 @@ def _get_indexes(segment):
             ):
                 first_whitespace_idx = fname_idx
 
+        for seg in context.siblings_post:
+            if seg.is_type("from_clause"):
```

Comment:
Nitpick: Would it be clearer to change this to `if seg is from_segment:` ?

Reply:
I've implemented this change 👍